src/robot.js

```diff
@@ -516,8 +516,10 @@ class Robot {
   // Returns nothing.
   parseHelp (path) {
     const scriptDocumentation = {}
-    const body = fs.readFileSync(path, 'utf-8')
-    const lines = body.split('\n')
+    const body = fs.readFileSync(require.resolve(path), 'utf-8')
```

Comment:
before, people would pass paths with the file extension. As we want to be compatible with both JavaScript and CoffeeScript, we need to accept paths that can be required without the file extension. But for parsing the docs we need it again, which is exactly what `require.resolve` does.